docs/enterprise-guide/enterprise-on-premises-guide/support/installation-support/script/op-support-data.sh

```diff
@@ -116,76 +114,116 @@ echo Network on localhost
 ip a
 echo "---"
 
+#CHECK LOCAL MEMCACHED CONFIG
+echo
+echo Search for memcached
+
+hash memcached 2>/dev/null
+if [ $? == 1 ]; then
+  echo "memcached is NOT installed"
+else
+  echo "memcached is installed"
+fi
+echo "---"
+
+#CHECK DOCKER ENVIRONMENT
+hash docker 2>/dev/null
+if [ $? == 1 ]; then
+  echo
+  echo "Docker is not installed yet, if you consider using the OpenProject containers, please install docker."
+else
+  docker ps -a
+  docker volume ls
+fi
 
+#SSL/TLS CERTS AVAILABLE
+if [ "$ownwebserver" != "${ownwebserver#[Nn]}" ]; then
+  echo Search for SSL certificate $ssl_certificate
+  find $ssl_certificate
+  echo Search for SSL key $ssl_key
+  find $ssl_key
+  openssl x509 -in $ssl_certificate -noout -text
+  openssl rsa -in $ssl_key -noout -text
+  openssl x509 -noout -modulus -in $ssl_certificate | openssl md5
+  openssl rsa -noout -modulus -in $ssl_key | openssl md5
+fi
+echo
+echo
+echo "========="
+echo " SUMMARY "
+echo "========="
+echo
 #CHECK WEBSERVER ON LOCALHOST PORTS 80 AND 443
-echo 'Checking Port 80,443 on IP 127.0.0.1 reachable? (succeeded = reachable)'
-netcat -z -v 127.0.0.1 80 2>&1
-netcat -z -v 127.0.0.1 443 2>&1
+echo 'Checking Port 80,443 on IP 127.0.0.1 reachable? (0=YES / 1=NO)'
+#netcat -z -v 127.0.0.1 80 2>&1
+echo 2>/dev/null > /dev/tcp/127.0.0.1/80 ; echo $?
```

Comment:
You can use this `2>/dev/null` in line 241 too to suppress the openproject warnings while getting the version.

```
$openproject_fp run bundle exec rake version 2>/dev/null
```

I know, that's not part of this PR. But I saw you use this here, so I just wanted to point out, that it would make sense there too.

Reply:
there is no warning when you run the script, i am curious how to surpress the apt warning, maybe in the same way...